services-geojson/src/commonMain/kotlin/org/maplibre/geojson/model/CoordinateContainer.kt

```diff
@@ -0,0 +1,26 @@
+package org.maplibre.geojson.model
+
+import kotlinx.serialization.Polymorphic
+import kotlinx.serialization.Serializable
+
+/**
+ * Each of the s geometries which make up GeoJson implement this interface and consume a varying
+ * dimension of [Point] list. Since this is varying, each geometry object fulfills the
+ * contract by replacing the generic with a well defined list of Points.
+ *
+ * @param <T> a generic allowing varying dimensions for each GeoJson geometry
+ * @since 3.0.0
+ */
+@Serializable
+@Polymorphic
```

Comment:
TIL ([docs](https://kotlinlang.org/api/kotlinx.serialization/kotlinx-serialization-core/kotlinx.serialization/-polymorphic/))

Reply:
Was also new to me, and in some step necessary. But now I am not sure anymore if this left by my converting. As the documentation says interfaces holding this automatically.